packages/backend/server/src/plugins/copilot/context/types.ts

```diff
@@ -0,0 +1,78 @@
+import { File } from 'node:buffer';
+
+import { z } from 'zod';
+
+import { parseDoc } from '../../../native';
+
+export enum ContextFileStatus {
+  processing = 'processing',
+  finished = 'finished',
+  failed = 'failed',
+}
+
+export const ContextConfigSchema = z.object({
+  workspaceId: z.string(),
+  files: z
+    .object({
+      id: z.string(),
+      chunk_size: z.number(),
+      name: z.string(),
+      status: z.enum([
+        ContextFileStatus.processing,
+        ContextFileStatus.finished,
+        ContextFileStatus.failed,
+      ]),
+      blobId: z.string(),
+    })
+    .array(),
+  docs: z.string().array(),
```

Comment:
Docs and files are sorted in the order in which they are added by the user, so they should be stored in an array.